vendor_prefixed/sumup-sdk/Readers/Readers.php

```
@@ -0,0 +1,450 @@
+ * Request payload for ReadersTerminateCheckoutRequest.
+ *
+ * @package SumUp\Services
+ */
+class ReadersTerminateCheckoutRequest
+{
+    /**
+     * Create request DTO from an associative array.
+     *
+     * @param array<string, mixed> $data
+     */
+    public static function fromArray(array $data): self
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$data'. (undefined)

(UnusedFormalParameter)